_includes/scenarios/integrate/create-own-identityattribute.md

```diff
@@ -0,0 +1,68 @@
+This guide explains the end to end flow of creating an [IdentityAttribute]({% link _docs_integrate/data-model-overview.md %}#identityattribute) for your own Connector as an Integrator. Before you start, you should check out our IdentityAttribute introduction first.
+
+<!--- TODO: Insert link to "IdentityAttribute introduction" --->
+
+## Input for creating an IdentityAttribute
+
+To create an IdentityAttribute as an Integrator for your own Connector, you need to proceed as described in the [Create an Attribute]({% link _docs_use-cases/use-case-consumption-create-an-attribute.md %}) use case documentation using the following content:
+
+| Property    | Value                                                                                                    |
+| ----------- | -------------------------------------------------------------------------------------------------------- |
+| `@type`     | `"IdentityAttribute"`                                                                                    |
+| `owner`     | `"<Address of your Connector>"`                                                                          |
+| `validFrom` | `"<start of Attribute validity>"`                                                                        |
+| `validTo`   | `"<end of Attribute validity>"`                                                                          |
+| `value`     | Specify an [IdentityAttribute Value]({% link _docs_integrate/attribute-values.md %}#identity-attributes) |
+| `tags`      | `["<additional information 1>", ..., "<additional information m>"]`                                      |
+
+You need to replace the placeholders marked with `<...>` appropriately. In particular, it is necessary that you insert one of the available [IdentityAttribute Values]({% link _docs_integrate/attribute-values.md %}#identity-attributes) into the `value` property. You are not allowed to specify the Address of a Connector other than your own as the value for the `owner` property, as the [Create an Attribute]({% link _docs_use-cases/use-case-consumption-create-an-attribute.md %}) use case relates to the creation of Attributes for yourself and not for others. Note that the properties `validFrom`, `validTo` and `tags` are optional, so you can omit them.
```

Comment:
> Note that the properties `validFrom`, `validTo` and `tags` are optional, so you can omit them.

In the [other table](https://enmeshed.eu/integrate/data-model-overview#identityattribute) is all information (data type and optional or not). I think it's not good to write this in text form.

Reply:
Now that I have replaced the table with a JSON payload, I have the following general question: 

If I give an example JSON payload and want to emphasise that some of the properties are optional and can be omitted, would you find it okay to point this out in text form then? With the tables, I had the option of adding an extra "Required" column, with the JSON payloads it is more difficult to indicate directly whether a property is optional.